sdk/tables/ci.yml

```diff
@@ -14,8 +14,7 @@ trigger:
 pr:
   branches:
     include:
-      # TODO: re-enable master pr validation after tables tests are ready
-      #- master
+      - master
```

Comment:
Unnecessary to disable/enable PR triggers in your fork, the trigger rules are actually controlled by the target branch anyway, so I suspect this didn't have an impact ;)

Reply:
This did disable `js - tables - ci` in PR validation. See this PR which still has this commented out: https://github.com/Azure/azure-sdk-for-js/pull/10207 it doesn't trigger `js - tables - ci`